suitcase/hdf5.py

```diff
@@ -96,25 +96,23 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
                     data = np.array(data)
-                    try:
+                    if data.dtype.kind in ['S', 'U']:
```

Comment:
Yes, it is from hxn. 

Please try runid=13123 and data_key='merlin1_stats1_asyn_pipeline_config'